### ISSUE TITLE
Password Reset Error.[BUG] 

### ISSUE THREAD

**Issue by AkameOda** [massif-press/compcon#2351]
**Describe the bug**
When trying to reset my password I reach an error when submitting the new password. "Unable to change password: Username/client id combination not found." It did not ask for a username (nor should it need to), only an email address. I used the latest code emailed to me with each attempt and produced a normal alphanumeric password & tried this without special characters, no change. I am on a different laptop as the PC I used to create the account is currently in need of repair.

**To Reproduce**
Attempt the password reset process using your email address, no username. Use a Gmail address. Try doing so on a different device that the one used to create the account.

**Expected behavior**
I hoped my password would be reset without error.

**Screenshots**
![compcon PW error](https://github.com/massif-press/compcon/assets/153905662/35dccb1f-8c54-4d11-9277-88026e1a1efb)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Microsoft Edge
 - Version: 2.3.19b

**Additional context**

**Comment by ApprenticeofEnder** [comment]
Can you try and reproduce the behaviour again, but this time capture the network request when you submit the actual password reset in dev tools? I want to see exactly what sort of response the server is giving (should be a cognito domain).

**Comment by AkameOda** [comment]
I've never opened Dev Tools on purpose before. Does this help or is it under the <head> section.
![PW Reset Dev tools](https://github.com/massif-press/compcon/assets/153905662/2022d712-c03a-4c30-9c65-b37d7221ffd6)

**Comment by AkameOda** [comment]
Not closed

**Comment by ApprenticeofEnder** [comment]
> I've never opened Dev Tools on purpose before. Does this help or is it under the section. ![PW Reset Dev tools](https://private-user-images.githubusercontent.com/153905662/293469647-2022d712-c03a-4c30-9c65-b37d7221ffd6.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQyNjc1NDQsIm5iZiI6MTcwNDI2NzI0NCwicGF0aCI6Ii8xNTM5MDU2NjIvMjkzNDY5NjQ3LTIwMjJkNzEyLWMwM2EtNGMzMC05YzY1LWIzN2Q3MjIxZmZkNi5KUEc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwM1QwNzM0MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMTdiYjhhYWUxODdhY2VmYTVhMmRiODE0M2ZmMDY5MDg5ZDRjZDc4NGVkMWFhNzE5ODBhY2VkNjlkMTY5MmU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.T1k2YAPaPzM4iYh5MUq0l41yiXI2VWoUw0pGRXg70gI)

No, you see the little wifi symbol at the top? That's what we'd be after, that's the network tab. Open that, get to the part where you click on the "submit password" button, and then look at the last entry in the last.

**Comment by AkameOda** [comment]
OK, here's the last 2 entries, since one is a failure & the last seems to be only a successful return of the error message.
![Debug 1](https://github.com/massif-press/compcon/assets/153905662/dfc1da82-550d-44ec-9cf7-c2704e5d61f2)
![Debug 2](https://github.com/massif-press/compcon/assets/153905662/916b7695-e33f-4aef-952f-2ff3d56acb73)

**Comment by AkameOda** [comment]
Full header on error/bad request.
![error header](https://github.com/massif-press/compcon/assets/153905662/cb1628df-cb31-4cd6-add2-3ea318c3f269)
![error header2](https://github.com/massif-press/compcon/assets/153905662/c71e7122-3887-42a5-9ab9-b1de77a9790c)

**Comment by ApprenticeofEnder** [comment]
Sorry for the delay, were you able to capture the actual response under the "response" tab for the one that returned an error?

**Comment by AkameOda** [comment]
{"__type":"UserNotFoundException","message":"Username/client id combination not found."}

**Comment by msprijatelj** [comment]
Okay my suggestions:
1. Make sure any browser extensions (uBlock, PrivacyBadger) that block elements whitelist all of CompCon’s connections before trying again.  
2. When resetting your password, try capitalizing your email address (Person@mail.com instead of person@mail.com). Legacy accounts allowed for case-sensitive emails; that bug has since been fixed.

**Comment by jarena3** [comment]
resolved in v3
